### ISSUE TITLE
Sensor recorder app sending null data

### ISSUE THREAD

**Issue by vivienlouradour** [vivienlouradour/weather-station#5]
Sensor recorder application is sending null record at night.  
Last valid record sent : 23:52
First valid record sent : 10:04

Api log sample :  
```
2018-07-01 21:52:45.401 [INFO]    posting record : Date : 07/01/2018 23:52:45 ; Temp : 30.53158 ; Hum : 37.84232 ; Broadcaster : pine64  
2018-07-01 21:52:45.424 [INFO]    Record created.  
2018-07-01 22:02:47.540 [INFO]    posting record :   
2018-07-01 22:02:47.541 [INFO]    Record is null (not created).  
  
[...]  
  
2018-07-02 07:54:47.106 [INFO]    posting record :   
2018-07-02 07:54:47.106 [INFO]    Record is null (not created).  
2018-07-02 08:04:49.139 [INFO]    posting record : Date : 07/02/2018 10:04:49 ; Temp : 26.42387 ; Hum : 40.87119 ; Broadcaster : pine64  
2018-07-02 08:04:49.165 [INFO]    Record created.  
```

**Comment by vivienlouradour** [comment]
Should be in SensorRecorder, file ![main.c](https://github.com/vivienlouradour/weather-station/blob/f6d1ce654091e50055a710a5fbbe4ebf0f1f4590/WeatherStation.SensorRecorder/main.c) :  

```
line 165 :  
sprintf(date, "%d-%02d-%02dT%d:%02d:%02d.000Z", tm.tm_year + 1900, tm.tm_mon + 1, tm.tm_mday, tm.tm_hour, tm.tm_min, tm.tm_sec);  
```
Replace `%d` by `%02d` for hour formatting.

**Comment by vivienlouradour** [comment]
Fixed : commit [58a873c](https://github.com/vivienlouradour/weather-station/commit/58a873cbe4d5cd0fc8c170104468415634d55834)
